src/dash-table/components/CellInput/index.tsx

```diff
@@ -47,6 +47,11 @@ export default class CellInput extends PureComponent<ICellProps, ICellState> {
             value
         } = this.props;
 
+        // input does not handle `null` correct (causes console error)
+        const sanitizedValue = this.state.value === null ?
```

Comment:
null is a valid value in the table but the input does not handle it well -- kind of annoying that JSON supports what the other does not